Cargo.lock

```diff
@@ -0,0 +1,137 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
+[[package]]
+name = "addr2line"
+version = "0.12.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "a49806b9dadc843c61e7c97e72490ad7f7220ae249012fbda9ad0609457c0543"
```

Comment:
Is this really necessary? Versions of all deps are specified using the fancy version language, and this should be enough. I think the Cargo.lock approach is bonkers. This package does not get many changes, so this doesn't matter so much, but if it did, either git history would be full of pointless noise or those versions would be outdated most of the time.

Reply:
(To avoid the same discussion for the nth time here's an from years ago I got on `cargo-update` regarding not including `Cargo.toml`: https://github.com/nabijaczleweli/cargo-update/issues/50; it hinges on specifying "maj.min" instead of just "maj" dependencies, but otherwise should be relevant here too)

Reply:
I was just following best practices ;) Also it is quite nice when updating some dependency to know which dependencies changed and how. But if you feel strongly against this, I can drop this part.

Reply:
> what you (I?) want is the "best" version of every dependency, without having it explicitly governed by some (possibly very outdated) "ground truth". 

Exactly. I think that "best practice" is misguided. So please drop that patch for now.

Reply:
done.